JarrydWorland/2021_Capstone_GP2_Team
# Instrumentation

## Description
This PR adds logging tools and seeds to level generation.

## Changes include
- Added level generation seed and logging categories. (594629d)
- Added additional log information and color support (2c3ba56)
- Added color to level generation log (0ebcd68)
- Added logging to EventManager (9763d0f)


## How to test
The logging filters can be found in the menu bar under `Utilities > Logging`, I am unsure as to the preferred behavior of the categories so for now they simply show a message if it has at least one of the enabled categories. So if you enable two categories both should show, but no others.

Level generation seed can be tested via `Debug Seed` field on the Level prefab. `0` will disable the seed and use a random value, providing a seed should force it each time the game is run.
